docs/UserGuide.md

```diff
@@ -263,6 +263,25 @@ To create a deadline named `event proposal` with due time at today's 6 pm.
 
 To create a deadline named `EE2024 homework 1` at 2016 1st October 6 am.
 
+### List tasks: `list`
+
+List certain type of task stored in the database
```

Comment:
> So why not the list command use these boxes to list events, deadlines and floating tasks all at once?
But for our first design, we do not even have to use `list`, since all the tasks will be shown automatically on the ui once they are created. And if the user wants to see tasks for a certain day, they could use `view DATE` command.